src/Slices/ServiceInventory/UI/Components/RowActionsMenu/RowActions.tsx

```diff
@@ -64,9 +62,6 @@ export const RowActions: React.FunctionComponent<InstanceActionsProps> = ({
     // -100 for each default option present above the first drilldown that is enabled.
     // diagnose and duplicate are always enabled, so this means it starts at -200
     let insetHeight = -200;
-    if (features && features.includes("instanceComposer") && !editDisabled) {
-      insetHeight = insetHeight - 100;
-    }
     if (!editDisabled) {
       insetHeight = insetHeight - 100;
```

Comment:
Right below it is the code that does the same, and deleted part required "instanceComposer" feature flag which won't be there from now on.

Reply:
yeah, no, that's the thing, previously, when you had the composer option present and enabled, it would deduce 100. Now the option is there by default, which means you'll have an issue with the offset in the dropdown if you open one of the drilldown menus. So you should deduce 200 in line 66, because now, we have two options that can be disabled/enabled. It's -100 for each enabled option.

Reply:
Oh, I misunderstood it, changed!